feeds.json

```diff
@@ -0,0 +1,321 @@
+          "url": "http://www.coolhunting.com/atom.xml"
+        },
+        {
+          "title": "Fashionista",
+          "url": "http://feeds.feedburner.com/fashionistacom"
+        },
+        {
+          "title": "Fashion Squad",
+          "url": "http://www.fashionsquad.com/feed/"
+        }
+      ]
+    },
+
+    {
+      "photography": [
+        {
+          "title": "500px",
+          "url": "http://feed.500px.com/500px-editors"
+        },
+        {
+          "title": "National Geographic: Photo of the Day",
+          "url": "http://www.nationalgeographic.com/rss/photography/photo-of-the-day"
+        },
+        {
+          "title": "The Big Picture",
+          "url": "http://www.boston.com/bigpicture/index.xml"
+        }
+      ]
+    },
+
+    {
+      "comics": [
+        {
+          "title": "XKCD",
+          "url": "http://xkcd.com/rss.xml"
+        },
+        {
+          "title": "The Oatmeal",
+          "url": "http://theoatmeal.com/feed/rss"
+        },
+        {
+          "title": "Dilbert Daily Strip",
+          "url": "http://feeds.feedburner.com/DilbertDailyStrip"
+        },
+        {
+          "title": "Ph.D. Comics",
+          "url": "http://www.phdcomics.com/gradfeed.php"
+        },
+        {
+          "title": "What If?",
+          "url": "http://what-if.xkcd.com/feed.atom"
+        }
+      ]
+    },
+
+    {
+      "science": [
+        {
+          "title": "National Geographic: News",
+          "url": "http://news.nationalgeographic.com/index.rss"
+        },
+        {
+          "title": "Scientific American: Global",
+          "url": "http://www.sciam.com/xml/sciam.xml"
+        },
+        {
+          "title": "TED Talks",
+          "url": "http://feeds.feedburner.com/tedtalks_video"
+        },
+        {
+          "title": "Science: Current Issue",
+          "url": "http://www.sciencemag.org/rss/current.xml"
+        },
+        {
+          "title": "Futurity.org",
+          "url": "http://www.futurity.org/feed/"
+        },
+        {
+          "title": "Popular Science",
+          "url": "http://www.popsci.com/rss.xml"
+        },
+        {
+          "title": "New York Times: Science",
+          "url": "http://rss.nytimes.com/services/xml/rss/nyt/Science.xml"
+        },
+        {
+          "title": "Io9",
+          "url": "http://feeds.gawker.com/io9/full"
+        }
+      ]
+    },
+
+    {
+      "finance": [
+        {
+          "title": "Get Rich Slowly",
+          "url": "http://www.getrichslowly.org/blog/feed/"
+        },
+        {
+          "title": "Freakonomics",
+          "url": "http://freakonomics.blogs.nytimes.com/feed/"
+        },
+        {
+          "title": "The Economist: The World This Week",
+          "url": "http://www.economist.com/rss/the_world_this_week_rss.xml"
+        },
+        {
+          "title": "Bloomberg",
+          "url": "http://www.newslookup.com/rss/business/bloomberg.rss"
+        }
+      ]
+    }
+  ]
+}
```

Comment:
67 feeds, if you were curious.

Taken from highly subscribed feeds (80k+) on feedly, using their categories as the topics (e.g. tech, gaming, finance)